libs/cache/cache.hpp

```diff
@@ -77,8 +85,8 @@ namespace iroha {
        * TODO 27/10/2017 luckychess Values are quite random and should be tuned
        * for better performance and may be even move to config IR-579
        */
-      const uint32_t MAX_HANDLER_MAP_SIZE_HIGH = 20000;
-      const uint32_t MAX_HANDLER_MAP_SIZE_LOW = 10000;
+      uint32_t max_handler_map_size_high_;
```

Comment:
Applicable to both vars:

1. `const` should be there
2. Awful and really long name
3. Would be nice to add doc comment